core/src/test/resources/TableMetadataV2MissingLastAssignedPartitionId.json

```diff
@@ -0,0 +1,71 @@
+{
+  "format-version": 2,
+  "table-uuid": "9c12d441-03fe-4693-9a96-a0705ddf69c1",
+  "location": "s3://bucket/test/location",
+  "last-sequence-number": 34,
+  "last-updated-ms": 1602638573590,
+  "last-column-id": 3,
```

Comment:
I just realized that this is `last-column-id` and not `last-assigned-column-id`. I think we should make the metadata property `last-partition-id` instead of `last-assigned-partition-id` to match. I'll merge this and we can fix it in a follow-up.